packages/salesforcedx-vscode-apex-debugger/README.md

```diff
@@ -1,5 +1,5 @@
 # Apex Debugger for Visual Studio Code
-This extension enables VS Code to use the real-time Apex Debugger with your scratch orgs.
+This extension enables VS Code to use the real-time Apex Debugger with your scratch orgs and to use ISV Customer Debugger.
```

Comment:
After reading completely thru, sounds like you may want to add "...Apex Debugger with your scratch orgs, or to use ISV Customer Debugger for sandboxes or DE orgs." Something like that.